lightly/embedding/_base.py

```diff
@@ -114,11 +115,21 @@ def init_checkpoint_callback(self,
                 Where to save the checkpoint.
 
         """
-        # initialize custom model checkpoint
-        self.checkpoint_callback = CustomModelCheckpoint()
-        self.checkpoint_callback.save_last = save_last
-        self.checkpoint_callback.save_top_k = save_top_k
-        self.checkpoint_callback.monitor = monitor
-
-        dirpath = self.cwd if dirpath is None else dirpath
-        self.checkpoint_callback.dirpath = dirpath
+
+        if pl.__version__[:3] in ['1.0', '1.1', '1.2']:
+            # initialize custom model checkpoint
```

Comment:
Since this part is not covered by tests, I would maybe add a warning to tell the user that support for `pl<1.3` will be dropped in the near future.

Reply:
and then we can remove the hack in 2-3 releases